packages/start/config/index.js

```diff
@@ -37,6 +36,35 @@ function solidStartServerFsRouter(config) {
       app
     );
 }
+const SolidStartServerFnsPlugin = createTanStackServerFnPlugin({
+  // This is the ID that will be available to look up and import
+  // our server function manifest and resolve its module
+  manifestVirtualImportId: "solidstart:server-fn-manifest",
+  client: {
+    getRuntimeCode: () =>
+      `import { createServerReference } from "${normalize(
+        fileURLToPath(new URL("../dist/runtime/server-runtime.js", import.meta.url))
```

Comment:
Is it intended to import a `server-runtime` in the client plugin 🤔? Or is this just a suboptimal naming 😅?

Reply:
(Nevermind, seems to be the naming 😅)

Reply:
tried to track through it and came to the same confusing conclusion